### PR TITLE
chore(): remove method deprecated 

### PR DESCRIPTION
**Pull request by Tony133**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
The method `isObject` is deprecated (see screenshot)

Schermata 2022-09-30 alle 15 17 26

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information
PR to be evaluated

### CODE REVIEW

File: lib/compiler/plugins-loader.ts [hunk 1]
```diff
@@ -31,7 +30,9 @@ export interface MultiNestCompilerPlugins {
 export class PluginsLoader {
   public load(plugins: PluginEntry[] = []): MultiNestCompilerPlugins {
     const pluginNames = plugins.map((entry) =>
-      isObject(entry) ? (entry as PluginAndOptions).name : (entry as string),
+      entry !== null && typeof entry === 'object'
+        ? (entry as PluginAndOptions).name
+        : (entry as string),
```
Comment (kamilmysliwiec) [inline lib/compiler/plugins-loader.ts:35]:
```suggestion
      entry ? (entry as PluginAndOptions)?.name : (entry as string)
```
?

Comment (micalevisk) [reply lib/compiler/plugins-loader.ts:35]: @Tony133 :ping_pong:

Comment (micalevisk) [reply lib/compiler/plugins-loader.ts:35]: @Tony133  :ping_pong:

File: lib/compiler/plugins-loader.ts [hunk 2]
```diff
@@ -63,9 +64,10 @@ export class PluginsLoader {
       if (!plugin.before && !plugin.after && !plugin.afterDeclarations) {
         throw new Error(CLI_ERRORS.WRONG_PLUGIN(pluginNames[index]));
       }
-      const options = isObject(plugins[index])
-        ? (plugins[index] as PluginAndOptions).options || {}
-        : {};
+      const options =
+        plugins[index] !== null && typeof plugins[index] === 'object'
+          ? (plugins[index] as PluginAndOptions).options || {}
+          : {};
```
Comment (micalevisk) [inline lib/compiler/plugins-loader.ts:70]:
```suggestion
      const options =
        plugins[index]?.options || {}
```

I guess?

File: lib/compiler/plugins-loader.ts (hunk 1 above)
Comment (Tony133) [reply lib/compiler/plugins-loader.ts:35]: Oops, I must have missed the notification for this PR 😅, sorry  🙏🏻

File: lib/compiler/plugins-loader.ts (hunk 2 above)
Comment (Tony133) [reply lib/compiler/plugins-loader.ts:68]: Yes, here too we can improve what I had done by writing a little less code. If it's okay for Kamil, I'll proceed with the change 👍

**Pull request merged**